flexmeasures/data/models/reporting/tests/test_aggregator.py

```diff
@@ -156,24 +156,58 @@ def test_resampling(setup_dummy_data):
     )
 
 
-def test_multiple_sources(setup_dummy_data):
-    """check that it uses the first source as default, instead of using all of them.
-    The result should contain only 1 belief per event.
+def test_source_transition(setup_dummy_data):
+    """The first 13 hours of the time window "belong" to Source 1 and are filled with 1.0.
+    From 12:00 to 24:00, there are events belonging to Source 2 with value -1.
+
+    We expect the reporter to use only the values defined in the `sources` array in the `input` field.
+    In case of encountering more that one source per event, the first is prioritized.
```

Comment:
The first, as in *the first source defined in the `sources` array*, or as in *the first source to have reported something about that event (oldest `belief_time`)*? Please clarify in the docstring.